src/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
```

Comment:
The card width should be 200px, as mentioned in the task description. Change the width to 200px.
And use `box-sizing: border-box` which automatically includes border width to card width